### DISCUSSION TITLE
How to install pyscript locally

### DISCUSSION THREAD

**Discussion by frchalaoux** [pyscript/pyscript#1208] [Q&A]
Hi All,

What is the best tutorial to install locally pyscript, pyodide and all the related stuff to get my development environment quick to load ? I often have no internet connexion in my travels

François-Régis

**Comment by xDRL64** [comment]
https://github.com/basvandertol/pyodide/releases/tag/localpyodide-v0.1

ça va te couter 500Mo sur le disque juste pour ça. J'essaie encore de trouver comment compiler mon main.py en main.wasm
mais personne ne semble vouloir montrer un simple exemple de comment faire ça............

**Reply by frchalaoux** [reply]
Thanks for this info. I'll have a look !

Did you try ? : https://github.com/pyscript/pyscript/discussions/1207

**Reply by xDRL64** [reply]
The release contains an example .html, yep that seems to work.

**Reply by xDRL64** [reply]
Sorry I didn't see the link : #1207 you wrote.
(and thought you asked me about the link I sent)
Thank you for information too.

**Comment by marimeireles** [comment]
Hey, I'm wondering if we should close this discussion due to the already opened issue: https://github.com/pyscript/pyscript/issues/581
In the issue you can find links to a description on how to do this Paul added a while ago.
IRC @tedpatrick also just went to this process.

**Reply by tedpatrick** [reply]
I thought I should share my pyscript-local development environment. I will revisit this.

**Reply by xDRL64** [reply]
Thank you Mariana, I'll see this.
And thank you Ted for sharing.

**Comment by chenchen100** [comment]
Download Pyolide and modify the src attribute of [[interpreters]] to local in py-config.toml

>[[interpreters]]
>src = "./pyodide/pyodide.js"
>#src = "https://fastly.jsdelivr.net/pyodide/v0.22.1/full/pyodide.js"
I will modify the icode on pyodide.js and pyodide.wasm.js earlier, https://jsliver...pyodide.js It seems like this, ensuring that it is local, but later it was found to be redundant. Just modify the py configuration and it will automatically connect to the local instead of the resources hosted on the network

Alternatively, you can use `vite` to publish resources and also change the src attribute, such as http://127.0.0.1:8080/pyodide/v21.3/full/pyodeide.js
